docs/cadt_rpc_api.md

```diff
@@ -20,52 +20,48 @@ If using a `CADT_API_KEY` append `--header 'x-api-key: <your-api-key-here>'` to
   - [POST Examples](#post-examples)
```

Comment:
I'd like to add to the preamble at the start of the doc. Feel free to word smith

When using this guide - it is important to note that the paradigm CADT uses is that all data updates using RPCs first go into a local "staging" table, i.e. not shared with the rest of the world. When using STAGING RPCs, you move data from this local "staging" table, to being updated on the blockchain, making the data public. 

Users should remember to first use PROJECTS, UNITS, etc. RPCs to stage data, then use STAGING RPCs to commit that data to chain.

Reply:
noted